specification/agricultureplatform/AgriculturePlatform.Management/tspconfig.yaml

```diff
@@ -27,6 +27,10 @@ options:
     flavor: "azure"
     generate-test: true
     generate-sample: true
+  "@azure-tools/typespec-rust":
```

Comment:
Do all these SDK successfully generate currently? @raych1 are there any plans to have an SDK validation pipeline for rust?

@LarryOsterman @heaths we should probably avoid adding all these configurations until we have some sort of validation pipeline for them.

Reply:
> are there any plans to have an SDK validation pipeline for rust?

I heard from @ronniegeraghty several months ago for the interest to onboard Rust to the `SDK Validation` pipeline. However, I haven't noticed any follow-ups yet.  

@LarryOsterman @heaths do we want to onboard Rust to the `SDK Validation` pipeline?

Reply:
We have individual pipeline to support the batch run on multiple specs. The config changes do not have to be in the `main` branch. The pipeline can be triggered against a feature branch.